src/schema/v2/HeroUnit/heroUnitsConnection.ts

```diff
@@ -19,22 +19,7 @@ export const heroUnitsConnection: GraphQLFieldConfig<void, ResolverContext> = {
     },
   }),
   resolve: async (_root, args, context) => {
-    const {
-      authenticatedHeroUnitsLoader,
-      heroUnitsLoader,
-      matchHeroUnitsLoader,
-    } = context
-
-    const { term } = args
-
-    if (term && !matchHeroUnitsLoader)
```

Comment:
FWIW we should make sure to _only_ guard against the unauthenticated loader access up above. If we're missing `term`, explicitly error against that (though that shouldn't be necessary since we can make term non-nullable)